s3/src/main/scala/akka/stream/alpakka/s3/impl/HttpRequests.scala

```diff
@@ -21,6 +20,33 @@ case class MetaHeaders(headers: Map[String, String])
 
 private[alpakka] object HttpRequests {
 
+  def listBucket(
+      bucket: String,
+      region: String,
+      prefix: Option[String] = None,
+      continuation_token: Option[String] = None
+  )(implicit conf: S3Settings): HttpRequest = {
+    val uri: Uri = {
+      val prefixString = if (prefix.isDefined) "&prefix=" + prefix.get else ""
+      val continuation_token_string =
+        if (continuation_token.isDefined)
+          "&continuation-token=" + continuation_token.get.replaceAll("=", "%3D").replaceAll("[+]", "%2B")
```

Comment:
This should use Uri-methods from http://doc.akka.io/docs/akka-http/current/scala/http/common/uri-model.html so we don't have to use .replaceAll. Will fix asap